engine/api/workflow/dao_run.go

```diff
@@ -41,6 +41,7 @@ type LoadRunOptions struct {
 	WithTests               bool
 	WithLightTests          bool
 	WithVulnerabilities     bool
+	WithoutDeleted          bool
```

Comment:
To prevent returns deleted runs, maybe we should have a WithDeleted flag instead of Without. So the default case will be to return only not deleted runs. No ?